Pipfile

```diff
@@ -15,13 +18,15 @@ drf-nested-routers = ">=0.90.0"
 django-filter = ">=1.1.0"
 requests = ">=2.18.0"
 django-tenant-schemas = "==1.9.0"
-django-cors-headers = "==2.2.0"
+django-cors-headers = "==2.4.0"
 querystring-parser = ">=1.2.3"
-djangorestframework-csv = "==1.3.3"
+djangorestframework-csv = "==2.1.0"
```

Comment:
This is a big version change. Any reason we were so far behind to start?

Reply:
Looking at the release history here: https://github.com/mjumbewu/django-rest-framework-csv/tags

It looks like I grabbed the latest version that is marked in github as a "release", but they've been tagging newer releases without actually using github's release mechanism.

So, let's do the update and I'll double-check that the CSV feature continues to function.